src/twisted/conch/insults/insults.py

```diff
@@ -431,23 +431,7 @@ def log(s):
     "CONTROL",
 )
 
-
-class _const:
-    """
-    @ivar name: A string naming this constant
-    """
-
-    def __init__(self, name: str) -> None:
-        self.name = name
-
-    def __repr__(self) -> str:
-        return "[" + self.name + "]"
-
-    def __bytes__(self) -> bytes:
-        return ("[" + self.name + "]").encode("ascii")
-
-
-FUNCTION_KEYS = [_const(_name).__bytes__() for _name in _KEY_NAMES]
+FUNCTION_KEYS = [f"[{_name}]".encode("ascii") for _name in _KEY_NAMES]
```

Comment:
I am going to skip over the compatibility concern here since I'm assuming the other reviewers did some due diligence here (and I see there was already a regression, so we are in uncomfortable territory).  In general though this type of type-change on a public attribute would need to be accompanied by some kind of transition period.